bci_tester/data.py

```diff
@@ -233,35 +263,42 @@ def create_BCI(
     )
 
 
-BASE_CONTAINER = create_BCI(
-    build_tag=f"bci/bci-base:{OS_VERSION}",
-    image_type="kiwi",
-    bci_type=ImageType.OS,
-)
+if OS_VERSION == "tumbleweed":
+    BASE_CONTAINER = create_BCI(
+        build_tag="tumbleweed:latest",
+        image_type="kiwi",
+        bci_type=ImageType.OS,
+    )
+else:
+    BASE_CONTAINER = create_BCI(
+        build_tag=f"bci/bci-base:{OS_CONTAINER_TAG}",
+        image_type="kiwi",
+        bci_type=ImageType.OS,
+    )
 MINIMAL_CONTAINER = create_BCI(
-    build_tag=f"bci/bci-minimal:{OS_VERSION}",
+    build_tag=f"bci/bci-minimal:{OS_CONTAINER_TAG}",
     image_type="kiwi",
-    available_versions=["15.4", "15.5"],
+    available_versions=ALLOWED_BASE_OS_VERSIONS,
     bci_type=ImageType.OS,
 )
 MICRO_CONTAINER = create_BCI(
-    build_tag=f"bci/bci-micro:{OS_VERSION}",
+    build_tag=f"bci/bci-micro:{OS_CONTAINER_TAG}",
     image_type="kiwi",
-    available_versions=["15.4", "15.5"],
+    available_versions=ALLOWED_BASE_OS_VERSIONS,
     bci_type=ImageType.OS,
 )
 BUSYBOX_CONTAINER = create_BCI(
-    build_tag=f"bci/bci-busybox:{OS_VERSION}",
+    build_tag=f"bci/bci-busybox:{OS_CONTAINER_TAG}",
     image_type="kiwi",
-    available_versions=["15.4", "15.5"],
+    available_versions=ALLOWED_BASE_OS_VERSIONS,
```

Comment:
There is no busybox in `15.3`
```suggestion
    available_versions=["15.4", "15.5", "tumbleweed"],
```